scripts/pkg/build_templates/opensearch/opensearch.rpm.spec

```diff
@@ -36,10 +36,17 @@ For more information, see: https://opensearch.org/
 cd %{_topdir} && pwd
 # Create necessary directories
 mkdir -p %{buildroot}%{pid_dir}
+mkdir -p %{buildroot}%{product_dir}/plugins
 # Install directories/files
 cp -a etc usr var %{buildroot}
 chmod 0755 %{buildroot}%{product_dir}/bin/*
-chmod 0755 %{buildroot}%{product_dir}/plugins/opensearch-security/tools/*
+if ls %{buildroot}%{product_dir}/plugins | grep opensearch-security; then
+    chmod 0755 %{buildroot}%{product_dir}/plugins/opensearch-security/tools/*
+fi
+# Pre-populate the folders to ensure rpm build success even without all plugins
+mkdir -p %{buildroot}%{config_dir}/opensearch-observability
+mkdir -p %{buildroot}%{config_dir}/opensearch-reports-scheduler
+mkdir -p %{buildroot}%{product_dir}/performance-analyzer-rca
```

Comment:
Great work! This will allow developers to speed up testing by commenting out most plugins.